onnxruntime/python/tools/quantization/quant_utils.py

```diff
@@ -112,12 +118,37 @@ def from_string(format):
 ONNX_TYPE_TO_NP_TYPE = {
     onnx_proto.TensorProto.INT8: numpy.dtype("int8"),
     onnx_proto.TensorProto.UINT8: numpy.dtype("uint8"),
+    onnx_proto.TensorProto.INT16: numpy.dtype("int16"),
+    onnx_proto.TensorProto.UINT16: numpy.dtype("uint16"),
     onnx_proto.TensorProto.FLOAT8E4M3FN: float8e4m3fn,
 }
 
+ONNX_INT_TYPE_RANGE = {
+    onnx_proto.TensorProto.UINT8: (0, 255),
+    onnx_proto.TensorProto.INT8: (-128, 127),
+    onnx_proto.TensorProto.UINT16: (0, 65535),
+    onnx_proto.TensorProto.INT16: (-32768, 32767),
+}
+
+ONNX_INT_TYPE_SYMMETRIC_RANGE = {
```

Comment:
That's right. Symmetric only really applies to signed types. This was done to make the implementation "simpler" to use since we can just use this dict with any int type.

That said, it would also be straightforward to not include unsigned types here. I'll make the change.

Reply:
Updated